### PR TITLE
store: Enclose column names in double quotes

### PR DESCRIPTION
**Pull request by tilacog**

### CODE REVIEW

File: store/postgres/src/relational_queries.rs [hunk 1]
```diff
@@ -2903,7 +2903,9 @@ fn write_column_names(column_names: &AttributeNames, table: &Table, out: &mut As
                 })
                 .peekable();
             while let Some(column_name) = iterator.next() {
+                out.push_sql("\"");
```
Comment (lutter) [inline store/postgres/src/relational_queries.rs:2906]: It would be better to do these as `out.push_identifier(column_name.as_str())?`

**Comment by leoyvens** [comment]
Are we able to test the problematic case?

**Review by lutter — APPROVED** [review]
Just change to `push_identifier` before committing.

**Comment by tilacog** [comment]
> Are we able to test the problematic case?

Sure. I'll write a test for a subgraph that has entities with fields named after SQL reserved words.

**Pull request merged**